api/apps/geoprocessing/src/modules/surface-cost/application/surface-cost-processor.spec.ts

```diff
@@ -82,43 +85,43 @@ describe(`when shapefile was converted to geojson`, () => {
 
     it(`should throw`, async () => {
       await expect(sut.process(getJob(scenarioId))).rejects.toThrow(/Missing/);
-      expect(puExtractor.mock).toHaveBeenCalledWith(getGeoJson());
+      expect(puExtractor.mock).toHaveBeenCalledWith(
+        getGeoJson(availablePlanningUnitIds),
+      );
     });
   });
 
   describe(`when cost was resolved`, () => {
+    const cost = getCost(availablePlanningUnitIds);
```

Comment:
minor, especially since this is a test, but I think this function and its use may be clearer if we called it `getCostSurface()` or something like this

Reply:
well we hit naming things again ; ) its already in `cost surface` context, so would adding it again deliver more value to reader?

Reply:
no problem in leaving as is as it's just a test, but well, we are returning a cost surface, not a cost: if going by context, should a reader assume we return the *cost of a cost surface*? but that's not even a thing in Marxan parlance, maybe `getSurface()` or `getCostByPlanningUnit()` would be more closely descriptive. bug again, not an issue in practice in this specific context.

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/225/commits/a6068c200c2bb6245a9b13cdd2fdb10f3134a821